migrations/900000000000021_ossf_scorecard.up.sql

```diff
@@ -0,0 +1,57 @@
+BEGIN;
+
+INSERT INTO mergestat.repo_sync_types (type, description, short_name, priority, type_group)
+VALUES ('OSSF_SCORECARD_REPO_SCAN', 'Executes an OSSF scorecard scan on a git repository', 'OSSF Scorecard Repo Scan', 3, 'GITHUB') ON CONFLICT DO NOTHING;
```

Comment:
Why is this made part of `GITHUB` `type_group`?

Reply:
You will see in the sync file this uses the GitHub credentials. We need it to share the same concurrency for that PAT. We were able to use the group here to account for that. We still need to take that into consideration for auto imports.

Reply:
👍